### PR TITLE
In 148172759 hide copy button on ie

### PR DESCRIPTION
**Pull request by aaronOSI**
Hide copy button for IE11.

Also fixing a spec helper that was flaking due to visibility / timing issues.

### CODE REVIEW

File: spec/support/helpers/datetime_helpers.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def mouse_select_datepicker(locator, date)
     zero_indexed_month = date.month - 1
     within locator do
       find('.rw-btn-calendar').click
-      find("[id$='cal_calendar__month_#{zero_indexed_month}-#{date.day}']").click
+      find("[id$='cal_calendar__month_#{zero_indexed_month}-#{date.day}']", visible: true).click
```
Comment (jgedeon-osi) [inline spec/support/helpers/datetime_helpers.rb]: Why was this added in as part of these changes?

File: app/assets/javascripts/components/screenings/HistoryCard.jsx [hunk 2]
```diff
@@ -75,7 +75,7 @@ export default class HistoryCard extends React.Component {
                 </div>
             }
           </div>
-          { !emptyInvolvements &&
+          { !emptyInvolvements && window.clipboardData === undefined &&
```
Comment (jgedeon-osi) [inline app/assets/javascripts/components/screenings/HistoryCard.jsx]: It wasn't clear from the story, but if this check is going to be permanent I think I would prefer it to be in a method/function call that gives a hint as to why it's there, something like 'disableForIe' or 'disableIfClipboardData'.

File: app/assets/javascripts/components/screenings/HistoryCard.jsx [hunk 3]
```diff
@@ -78,6 +78,7 @@ export default class HistoryCard extends React.Component {
           { !emptyInvolvements &&
             <div className='row'>
               <div className='centered'>
+                { window.clipboardData === undefined &&
```
Comment (NateOSI) [inline app/assets/javascripts/components/screenings/HistoryCard.jsx]: Hey is it possible to move (window.clipboardData === undefined) into config and call it something like browserIsIE? or IEClipboard? trying to isolate where we use window in our app if possible

File: spec/support/helpers/datetime_helpers.rb (hunk 1 above)
Comment (aaronOSI) [inline spec/support/helpers/datetime_helpers.rb]: @jgedeon-osi Not sure how this was able to work before, but a lot of specs were failing due to visibility/click issues
